showcase/tests/integration/components/hds/form/header/index-test.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'showcase/tests/helpers';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | hds/form/header/index', function (hooks) {
```

Comment:
[todo] we should also test that it yields the `Title` and `Description` children

Reply:
Added tests